docs/user_manual/source/xsecure.rst

```diff
@@ -141,6 +144,8 @@ For these registers a second copy of the register is added which stores a comple
 Control flow hardening
 ----------------------
 A hardware check is performed to check if branches are taken (or not taken) as they should.
+Jumps and branches stay two cycles in ID and EX to enable the pipeline to recompute the PC target and the branch decision. These recomputed values are then compared
```

Comment:
We do not need a separate 'control flow hardening' section. Can you merge its content into the 'hardened pc' section?

Reply:
Sure, I will push an update.